modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -47,11 +48,21 @@ test('can import a folder with document type restrictions and workflow', async (
 test('can import a lar file selecting some items to import', async ({
 	exportImportPage,
 }) => {
+	await exportImportPage.goToExport();
+
+	const exportName = 'MyExport-' + getRandomString();
+
+	await exportImportPage.createNewExportProcess(exportName);
+
+	expect(
```

Comment:
I think this can be improved like this:

```typescript
expect(await exportImportPage.page.getByText(exportName).locator('../..').getByText('Successful')).toBeVisible();
```

Reply:
For the record, as we discussed, the await should be in the beginning:

```java
	await expect(
		exportImportPage.page
			.getByText(exportName)
			.locator('../..')
			.getByText('Successful')
	).toBeVisible();
```

Reply:
Thanks for your comment :heart: